drivers/si70xx/si70xx.c

```diff
@@ -18,22 +18,34 @@
  * @}
  */
 
-
+#include <string.h>
 #include "xtimer.h"
 
+#include "si70xx_internals.h"
 #include "si70xx.h"
 
+#define ENABLE_DEBUG (0)
+#include "debug.h"
+
+#define SI70XX_I2C     (dev->params.i2c_dev)
+#define SI70XX_ADDR    (dev->params.address)
+
 /**
  * @brief   Utility method to perform and reconstruct a measurement.
  */
 static uint32_t si70xx_measure(const si70xx_t *dev, uint8_t command)
 {
     uint8_t result[2];
 
-    i2c_acquire(dev->i2c_dev);
-    i2c_write_byte(dev->i2c_dev, dev->address, command);
-    i2c_read_bytes(dev->i2c_dev, dev->address, result, 2);
-    i2c_release(dev->i2c_dev);
+    i2c_acquire(SI70XX_I2C);
+    
+    if (i2c_write_byte(SI70XX_I2C, SI70XX_ADDR, command) != 1) {
+        DEBUG("[ERROR] Cannot write command '%d' to I2C.\n", command);
+    }
+
+    if (i2c_read_bytes(SI70XX_I2C, SI70XX_ADDR, result, 2) != 2) {
+        DEBUG("[ERROR] Cannot read command '%d' result from I2C.\n", command);
+    }
 
     /* reconstruct raw result */
     return ((uint32_t)result[0] << 8) + (result[1] & 0xfc);
```

Comment:
as you do not return on error, what happens here - or what meaning does the value have then? Further why return `uint32_t` when `uint16_t` seems to be sufficient for `uint8_t result[2]`?